SphereEversion/ToMathlib/Analysis/InnerProductSpace/CrossProduct.lean

```diff
@@ -56,15 +56,11 @@ theorem inner_crossProduct_apply (u v w : E) : ⟪u×₃v, w⟫ = ω.volumeForm
              LinearMap.coe_comp, Function.comp_apply, LinearMap.llcomp_apply, LinearEquiv.coe_coe,
              LinearEquiv.trans_apply, LinearIsometryEquiv.coe_toLinearEquiv, LinearMap.coe_comp]
   rw [InnerProductSpace.toDual_symm_apply]
-  simp only [LinearMap.coe_toContinuousLinearMap', Function.comp_apply, LinearMap.llcomp_apply,
```

Comment:
Similarly for the three un-squeezing in this file?

Reply:
With `simp only`: 

```console
4.84s user 1.01s system 107% cpu 5.443 total
```

With `simp`:

```
5.13s user 1.11s system 115% cpu 5.418 total
```

Reply:
What do you think? Good enough?

Reply:
That's for the overall file, right? Yup, I'd call that fine. (For a single simp, 300ms or perhaps 500ms is where I start to wonder if squeezing makes sense. For now, I think that's fine.)

Reply:
Yep, whole file.